### PR TITLE
More visibility to chase test race condition

### PR DESCRIPTION
**Pull request by jcorbin**
- logs from resetAll
- logger out of allocCluster for tests to use themselves

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -524,6 +524,18 @@ TChannelConnection.prototype.resetAll = function resetAll(err) {
     var self = this;
     if (self.closing) return;
     self.closing = true;
+
+    var inOpKeys = Object.keys(self.inOps);
+    var outOpKeys = Object.keys(self.outOps);
+
+    self.logger.info('resetting all connections', {
```
Comment (jcorbin) [inline index.js]: may should be a warn

Comment (Raynos) [inline index.js]: consider warning if there is an err argument and assume that no err === clean shutdown and err === not clean shutdown.

**Comment by Raynos** [comment]
lgtm.

**Comment by jcorbin** [comment]
hacked in a level switch for now, would be better on a `logger.log(level, msg, extra)` api, but the null / echo loggers we hacked into TChannel don't suffice.  So leaving like this pending proper logtron-pocalypse.

**Pull request merged**